deny.toml

```diff
@@ -91,6 +91,7 @@ allow = [
     "BSD-3-Clause",
     "MPL-2.0",
     "Unicode-DFS-2016",
+    "Unicode-3.0",
```

Comment:
Added this because a cargo-deny check failed on this. It is a permissive MIT style license: 
- https://spdx.org/licenses/Unicode-3.0.html
- https://opensource.org/license/unicode-license-v3

Reply:
Note that that is a non-blocking job and that file is mostly managed in a template.  I seem to have missed pushing a template update which has now been addressed.

Reply:
Do you want me to remove it again?

Reply:
Yes, to avoid any weird merges.

Reply:
Done 👍